include/onnxruntime/core/graph/graph.h

```diff
@@ -780,6 +783,12 @@ class Graph {
   */
   void SetInputs(const std::vector<const NodeArg*>& inputs);
 
+  /** Explicitly update specified graph initializers.
+  @param updated_input_map An map contains the name of the initializer that needs to update and the
+  corresponding new value. This function will overwrite the old NodeArg with the new value.
+  */
+  void SyncWithFP16Initializer(const std::unordered_map<std::string, NodeArg*>& updated_input_map);
```

Comment:
>SyncWithFP16Initializer [](start = 7, length = 23)

Could this be a bit more generic than just FP16? For example, we might have BFP16 later?

Reply:
This method is not being used anymore. Deleted.

---
In reply to: [434893065](https://github.com/microsoft/onnxruntime/pull/4113#discussion_r434893065) [](ancestors = 434893065)